qbft/round_change.go

```diff
@@ -213,6 +214,7 @@ func validRoundChangeForData(
 	height Height,
 	round Round,
 	fullData []byte,
+	verifySignature bool,
```

Comment:
switch to dontVerifySignature?

Reply:
on another thought, maybe we should go for the validateX() and validateXNoVerify() pattern?

Reply:
Agree